mongo/integration/crud_helpers_test.go

```diff
@@ -1425,8 +1425,22 @@ func executeAdminCommand(mt *mtest.T, op *operation) {
 		return
 	}
 
+	rco := options.RunCmd()
+	rpVal, err := op.Arguments.LookupErr("readPreference", "mode")
```

Comment:
Oh good point. I had not considered that it requires exporting the type from the `unified` package. Some thoughts:

From grepping, I do not see other imports to `go.mongodb.org/mongo-driver/mongo/integration/unified`. Not that it is necessarily telling. I do not see a clear reason the `integration` package cannot depend on types from the nested `unified` package.

Parsing a read preference seems generically useful. Though it may be at odds with the test policy's "Duplication is OK" section.

I will defer to your judgement, but I am OK with this change or the prior parsing.

Reply:
Yeah looks like this is the only import of the unified package. The only issue I foresee with exporting `readPreference` is a possible import cycle (if the `unified` package were to import something from `integration`). Since this is not production code, I think it's ok if we cross that bridge when/if we get there.